doc/uploading-and-dowloading-files.md

```diff
@@ -0,0 +1,20 @@
+# Uploading and Downloading Files
+
+## Uploading files
+
+- to copy a file to the required remote environment:
+
+```
+cat RELATIVE-PATH/FILENAME.csv | cf ssh beis-roda-ENVIRONMENT -c "cat > FILENAME.csv"
+```
+
+Note: `cat` cannot create a directory. This command will fail if a path is passed to `cat >` . This also means that the copied file will be found in the **root** of the remote project; you will need to move the file to the required directory after copying. 
+
```

Comment:
Perhaps excessively nit-picky, but I don't think this is quite accurate -- I think the uploaded file ends up in `/root` which is the home directory (aka `~/`) of the logged-in user `root`:

```
$ cat local_foo.txt | cf ssh beis-roda-staging -c "cat > FILENAME.csv"
$ cf ssh beis-roda-staging                      
root@7f3df505-fd30-4569-50f2-a078:/app# cd ~
root@7f3df505-fd30-4569-50f2-a078:~# ls
FILENAME.csv
root@7f3df505-fd30-4569-50f2-a078:~# pwd
/root
root@7f3df505-fd30-4569-50f2-a078:~# cd /
root@7f3df505-fd30-4569-50f2-a078:/# ls
app  boot  dev			 etc   lib    media  opt   root  sbin  sys  usr
bin  deps  docker-entrypoint.sh  home  lib64  mnt    proc  run	 srv   tmp  var
root@7f3df505-fd30-4569-50f2-a078:/# pwd
/
```

Feel free to disregard if you want!

Reply:
No I think you're absolutely right and I will amend it